src/QMCWaveFunctions/RotatedSPOs.cpp

```diff
@@ -361,24 +421,60 @@ void RotatedSPOs::buildOptVariables(const RotationIndices& rotations, const Rota
     }
   }
 
+#ifdef QMC_COMPLEX
+  for (int i = 0; i < nparams_active; i++)
+  {
+    p = m_act_rot_inds_[i].first;
+    q = m_act_rot_inds_[i].second;
+    std::stringstream sstr;
+    sstr << my_name_ << "_orb_rot_" << (p < 10 ? "0" : "") << (p < 100 ? "0" : "") << (p < 1000 ? "0" : "") << p << "_"
+         << (q < 10 ? "0" : "") << (q < 100 ? "0" : "") << (q < 1000 ? "0" : "") << q << "_i";
+
+    // If the user input parameters, use those. Otherwise, initialize the parameters to zero
+    if (params_supplied_)
+    {
+      myVars.insert(sstr.str(), std::imag(params_[i]));
+    }
+    else
+    {
+      myVars.insert(sstr.str(), 0.0);
+    }
+  }
+#endif
```

Comment:
I feel this duplication can be much simplified if we put real and imag next to each other.